### DISCUSSION TITLE
Should we change the name of the repository or the organization to rudof?

### DISCUSSION THREAD

**Discussion by labra** [rudof-project/rudof#134] [Ideas]
At this moment the project is part of the [weso](https://www.weso.es/) organization which is mainly a research lab where I work at the University of Oviedo (WESO = WEb Semantics Oviedo). Although I am happy with it...I think it would probably make the project more scalable if we use `rudofproject` as a top level organization and we put inside it, `rudof` as a repository for `shapes-rs`. 

My main reason is that I think in the long term more people would be happier to collaborate or participate in an organization that is not specifically mentioning Oviedo in it's name than if we keep it inside `weso`...

Does anyone have an opinion in favour or against this change?

**Comment by angelip2303** [comment]
I believe this is an excellent approach, as it aligns well with best practices and better suits the project's intended scope. It will also help us maintain a more organized structure by defining each complementary repository within the `rudof` organization, rather than attaching them to the main WESO organization.

**Reply by labra** [reply]
Now the project is in its own organization rudof-project
